CRM/Contribute/BAO/Contribution.php

```diff
@@ -5457,4 +5457,32 @@ public static function createProportionalFinancialEntries($entityParams, $lineIt
     }
   }
 
+  /**
+   * Calculate Tax when Financial Type is changed.
+   *
+   * @param array $params
+   * @param float $totalAmount
+   * @param array $oldTaxAmounts
+   * @param float $changeFTAmount
+   *
+   */
+  public static function calculateTaxForChangeInFinancialType(
+    &$params, &$totalAmount,
+    &$oldTaxAmounts, &$changeFTAmount
+  ) {
+    $taxAmountAfterFTChange = self::calculateTaxAfterChangeInFinancialTypeForLineItems($params['line_item'], $params['contribution']->id);
```

Comment:
Why abbreviate Financial Type in some places and not in others? It would be nice to keep this consistent. My personal preference is to avoid abbreviations as much as possible